_includes/ref/bitcoin-core/rpcs/intro.md

```diff
@@ -9,153 +9,199 @@ http://opensource.org/licenses/MIT.
-JSON-RPC interface and how the command-line interface translates it.
-
-In order to start `bitcoind`, you will need to set a password for
-JSON-RPC in the `bitcoin.conf` file. See the [Examples
-Page][devexamples] for details. JSON-RPC starts on port 8332 for mainnet
-and 18332 for testnet and regtest. By default, `bitcoind` doesn't use a
-JSON-RPC user, but you can set one (see `bitcoind --help`).
-
-The interface is not intended for public access and is only accessible
-from localhost by default.
-
-RPCs are made using the standard JSON-RPC 1.0 syntax, which sends several
-standard arguments:
+Bitcoin Core provides a remote procedure call (RPC) interface for various
+administrative tasks, wallet operations, and queries about network and block
+chain data.
+
+If you start Bitcoin Core using `bitcoin-xt`, the RPC interface is disabled by
+default. To enable it, set `server=1` in `bitcoin.conf` or supply the `-server`
+argument when invoking the program.
+
+If you start Bitcoin Core using `bitcoind`, the RPC interface is enabled by
+default. To disable it, set `server=0` in `bitcoin.conf`.
+
+The interface requires the user to provide a password for authenticating RPC
+requests. This password can be set either using the `rpcpassword` property in
+`bitcoin.conf` or by supplying the `-rpcpassword` program argument. Optionally a
+username can be set using the `rpcuser` configuration value. See the [Examples
+Page][devexamples] for more information about setting Bitcoin Core configuration
+values.
+
+Open-source client libraries for the RPC interface are readily available in most
+modern programming languages, so you probably don't need to write your own from
+scratch. Bitcoin Core also ships with its own compiled C++ RPC client,
+`bitcoin-cli`, located in the `bin` directory alongside `bitcoind` and
+`bitcoin-qt`. The `bitcoin-cli` program can be used as a command-line interface
+(CLI) to Bitcoin Core or for making RPC calls from applications written in
+languages lacking a suitable native client. The remainder of this section
+describes the Bitcoin Core RPC protocol in detail.
+
+The Bitcoin Core RPC service listens for HTTP `POST` requests on port 8332 in
+mainnet mode or 18332 in testnet or regtest mode. The port number can be changed
+by setting `rpcport` in `bitcoin.conf`. By default the RPC service binds to your
+server's [localhost](https://en.wikipedia.org/wiki/Localhost) loopback
+network<!--noref--> interface so it's not accessible from other servers.
+Authentication is implemented using HTTP [basic
+authentication](https://en.wikipedia.org/wiki/Basic_access_authentication). RPC
+HTTP requests must include a `Content-Type` header set to `text/plain` and a
+`Content-Length` header set to the size of the request body.
+
+The format of the request body and response data is based on [version 1.0 of the
+JSON-RPC specification](http://json-rpc<!--noref-->.org/wiki/specification). Specifically,
```

Comment:
It's best to put links in the the `_includes/references.md` file and then reference them here.  For example, you would add this to the external links section of that file:

    [JSON-RPC 1.0]: http://json-rpc.org/wiki/specification

Then change this link to,

    The format of the request body and response data is based on [version 1.0 of the
    JSON-RPC specification][JSON-RPC 1.0]. Specifically,

(Note change from parenthesis to brackets.